app/data/dbConn.py

```diff
@@ -1,21 +1,24 @@
+from os import path
```

Comment:
why is code in data dir?
in example of games data dir would contain music, models, textures (data)
if you would play some mp3 to user on your website, you would keep that in data dir too
you keep code in data, so maybe rename data to code

Reply:
is name "src" also good ?

Reply:
sure, 'src' is from 'source'

Reply:
done, now is it correct to put folder utilities into src ? what about register_validation file? Can I put it also there?

Reply:
i would put all the code inside src